pkg/puller/puller.go

```diff
@@ -29,7 +29,7 @@ const loggerName = "puller"
 
 var errCursorsLength = errors.New("cursors length mismatch")
 
-const DefaultSyncErrorSleepDur = time.Second * 5
+const DefaultSyncErrorSleepDur = time.Second * 30
```

Comment:
I'm just curious as to why the length of sleep needs to be increased.

Reply:
it's an arbitrary value, 5 seconds seemed a bit aggressive to me after some thinking